cmd/createcluster.go

```diff
@@ -152,22 +147,30 @@ func runCreateCluster(ctx context.Context, w io.Writer, conf clusterConfig) erro
 		if err != nil {
 			return err
 		}
+
+		conf.FeeRecipientAddrs = def.FeeRecipientAddresses()
+		conf.WithdrawalAddrs = def.WithdrawalAddresses()
+
+		if strings.TrimSpace(def.Name) != "" {
+			conf.Name = def.Name
+		}
```

Comment:
why this conditional?

Reply:
removing since as you suggested, definition is source of truth